features/com.google.cloud.tools.eclipse.3rdparty.feature/feature.xml

```diff
@@ -78,7 +78,7 @@
          id="org.yaml.snakeyaml"
          download-size="0"
          install-size="0"
-         version="1.17"
+         version="1.17.0"
```

Comment:
I could swear there was a reason for this. Could you pull this file into a separate PR in case we need to roll this bit back? Thanks.